configure.ac

```diff
@@ -6572,6 +6572,35 @@ then
   [Define if you have struct stat.st_mtimensec])
 fi
 
+# Check if year with century should be normalized for strftime
```

Comment:
This comment is unneeded; the information is given in the line below.

Reply:
Oops I somehow requested for a re-review before I actually pushed my commit. The comment is now removed. Thanks.